modules/rest-api/pages/rest-xdcr-create-ref.adoc

```diff
@@ -32,6 +32,9 @@ curl -v -u [admin]:[password]
   -d password=[password]
 ----
 
+The `name` specified for the destination cluster is for local use on the _source_ cluster only; and so need not be identical to any name defined on the destination cluster.
+The `remote-host` can be an IP address (and if appropriate, can be an _alternate address_) or a hostname of a node on the destination cluster.
```

Comment:
hostname -> FQDN

Reply:
Hostname is still valid - so maybe hostname or FQDN

Reply:
I've changed it to: The remote-host should be the IP address (if appropriate, the alternate address), the hostname, or the fully qualified domain-name of a node on the destination cluster.